### PR TITLE
Spelling fixes

### PR DESCRIPTION
**Pull request by AngelOnFira**

### CODE REVIEW

**Comment by AngelOnFira** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #44** <a href="https://app.graphite.dev/github/pr/rivet-gg/site/44?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/rivet-gg/site/44?utm_source=stack-comment).

File: src/pages/privacy.mdx [hunk 1]
```diff
@@ -441,7 +441,7 @@ You can opt out of receiving ads that are personalized as served by our Service
 
 - The NAI's opt-out platform:{' '} [**http://www.networkadvertising.org/choices/**](http://www.networkadvertising.org/choices/)
 - The EDAA's opt-out platform{' '} [**http://www.youronlinechoices.com/**](http://www.youronlinechoices.com/)
-- The DAA's opt-out platform:{' '} [**http://optout.aboutads.info/?c=2&lang=EN**](http://optout.aboutads.info/?c=2&lang=EN)
+- The DATA's opt-out platform:{' '} [**http://optout.aboutads.info/?c=2&lang=EN**](http://optout.aboutads.info/?c=2&lang=EN)
```
Comment (AngelOnFira) [inline src/pages/privacy.mdx:444]: Probably false positive

Comment (NathanFlurry) [reply src/pages/privacy.mdx:444]: Yep, this needs to be reverted.

**Review by NathanFlurry — CHANGES_REQUESTED** [review]

File: src/pages/privacy.mdx (hunk 1 above)
Comment (AngelOnFira) [inline src/pages/privacy.mdx:444]:
```suggestion
- The DAA's opt-out platform:{' '} [**http://optout.aboutads.info/?c=2&lang=EN**](http://optout.aboutads.info/?c=2&lang=EN)
```

**Review by NathanFlurry — APPROVED** [review]

**Pull request merged**